### ISSUE TITLE
Cargo.lock is outdated on 0.3.0 tag

### ISSUE THREAD

**Issue by figsoda** [ouch-org/ouch#143]
Can you create a new 0.3.1 release with an up to date lock file?

**Comment by marcospb19** [comment]
Oh no, at which branch?

**Comment by figsoda** [comment]
https://github.com/ouch-org/ouch/blob/0.3.0/Cargo.lock#L296

**Comment by marcospb19** [comment]
Oof, that hurts, it took me some time to do that release, does this gives a problem with the `NixOS` packaging? I can make another release in one hour.

**Comment by figsoda** [comment]
Yes, I would have to patch the lock file

If you are making another release I will just skip packaging 0.3.0 and update directly to 0.3.1

**Comment by SpyrosRoum** [comment]
Little off topic but about releasing in general, don't forget to `cargo publish` as well!

**Comment by marcospb19** [comment]
> If you are making another release I will just skip packaging 0.3.0 and update directly to 0.3.1

Ok ok, I was asking exactly because I did not published in Crates.io yet, was a stupid mistake by me, I did not slept today, and it's already 1PM, maybe I'm a bit tired xD.

> Little off topic but about releasing in general, don't forget to cargo publish as well!

Yep, Vinícius thought I already had permissions to publish, I asked him, as he is the only owner in the https://crates.io/crates/ouch, I will publish there later today, the next releases will be synced I promise.

---

More on this bit of offtopic, I posted the 0.3.0 version release at https://www.reddit.com/r/rust/comments/ql51ff/ouch_030_released/, @epage asked if we have any feedback on `clap`, but you both have been using it more than me, so maybe yall should reply to him there if something comes to mind.
